src/workers/postAddedSlackChannelSend.ts

```diff
@@ -38,6 +39,24 @@ export const postAddedSlackChannelSendWorker: TypedWorker<'api.v1.post-visible'>
         ]);
         const source = await post.source;
 
+        let adminJoinToken: string | undefined;
+
+        if (source.type === SourceType.Squad) {
+          const admin = await con.getRepository(SourceMember).findOne({
+            where: {
```

Comment:
Do we have to add the select statement?

Reply:
for sure!

Reply:
Added in 30b47a9a41b25ba688cb2a5c58ac8d5914e674cf together with test